app/javascript/packages/captcha-submit-button/captcha-submit-button-element.ts

```diff
@@ -29,14 +29,26 @@ class CaptchaSubmitButtonElement extends HTMLElement {
     return this.getAttribute('recaptcha-action')!;
   }
 
+  get isRecaptchaEnterprise(): boolean {
+    return this.getAttribute('recaptcha-enterprise') === 'true';
+  }
+
+  get recaptchaClient(): ReCaptchaV2.ReCaptcha {
```

Comment:
How come we don't need to import a class/type for this?

Reply:
I think it's because of how the DefinitelyTyped definitions are automatically loaded, and this particular type is defined as a global namespace (?):

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a7da713fe6ebbb37d3d7128b46bf849436bac75b/types/grecaptcha/index.d.ts#L15